libraries/AC_Fence/AC_PolyFence_loader.cpp

```diff
@@ -212,35 +242,41 @@ bool AC_PolyFence_loader::breached(const Vector2f& pos_cm) const
     // check we are inside each inclusion zone:
     for (uint8_t i=0; i<_num_loaded_inclusion_boundaries; i++) {
         const InclusionBoundary &boundary = _loaded_inclusion_boundary[i];
-        if (Polygon_outside(pos_cm, boundary.points, boundary.count)) {
+        if (Polygon_outside(pos, boundary.lla_points, boundary.count)) {
             return true;
         }
     }
 
     // check we are outside each exclusion zone:
     for (uint8_t i=0; i<_num_loaded_exclusion_boundaries; i++) {
         const ExclusionBoundary &boundary = _loaded_exclusion_boundary[i];
-        if (!Polygon_outside(pos_cm, boundary.points, boundary.count)) {
+        if (!Polygon_outside(pos, boundary.lla_points, boundary.count)) {
```

Comment:
sorry to be annoying but in terms of naming, let's call it "points_lla".  normally we put units at the end of variable names and I see in other parts of the code "points_lla" is used.

Reply:
Ack - I'll update.